Suggestion form: JS error on first load.

The following error is printed in the developer console on first load of the page:

```require.js:900 Error: cannot call methods on tooltip prior to initialization; attempted to call method 'destroy'```

--- a/inspirehep/modules/literaturesuggest/static/js/literaturesuggest/literature_submission_form.js
+++ b/inspirehep/modules/literaturesuggest/static/js/literaturesuggest/literature_submission_form.js
@@ -155,6 +155,7 @@ define(function(require, exports, module) {
         .fieldsGroup({
           onEmpty: function enableProceedingsBox() {
             that.$nonpublic_note.removeAttr('disabled');
+            $('.tooltip-wrapper').tooltip(); // trigger the tooltip
             $('.tooltip-wrapper').tooltip('destroy'); // destroy the tooltip
           },
           onNotEmpty: function disableProceedingsBox() {

--- a/inspirehep/modules/literaturesuggest/templates/literaturesuggest/forms/suggest.html
+++ b/inspirehep/modules/literaturesuggest/templates/literaturesuggest/forms/suggest.html
@@ -53,6 +53,11 @@
 
 {% block javascript %}
 {{ super() }}
+
+{%- assets "inspirehep_literaturesuggest_js" %}
+<script src="{{ ASSET_URL }}"></script>
+{%- endassets %}
+
 <script type="text/javascript">
 require(
   [
@@ -97,8 +102,4 @@ require(
   });
 });
 </script>
-
-{%- assets "inspirehep_literaturesuggest_js" %}
-<script src="{{ ASSET_URL }}"></script>
-{%- endassets %}
 {% endblock javascript %}

--- a/inspirehep/modules/theme/bundles.py
+++ b/inspirehep/modules/theme/bundles.py
@@ -78,7 +78,8 @@ detailedjs = NpmBundle(
 )
 
 css = NpmBundle(
-    "scss/inspirehep.scss",
+    "node_modules/angular-loading-bar/build/loading-bar.css",
+    "scss/inspirehep.scss",  # Keep last to override previous stylesheets
     filters="scss, cleancss",
     output="gen/inspirehep.%(version)s.css",
     depends="scss/**/*.scss",

--- a/inspirehep/modules/theme/static/scss/inspirehep.scss
+++ b/inspirehep/modules/theme/static/scss/inspirehep.scss
@@ -56,6 +56,4 @@ $fa-font-path: "/static/node_modules/font-awesome/fonts";
 
 @import "../node_modules/toastr/toastr";
 
-@import "../node_modules/angular-loading-bar/build/loading-bar.css";
-
 @import "../node_modules/impact-graphs/style";

